modules/EvseV2G/charger/ISO15118_chargerImpl.cpp

```diff
@@ -353,6 +294,32 @@ void ISO15118_chargerImpl::handle_stop_charging(bool& stop) {
     }
 }
 
+void ISO15118_chargerImpl::handle_set_physical_values(types::iso15118_charger::SetupPhysicalValues& physical_values) {
+    // FIXME IMPLEMENT ME CORRECTLY: can we just update those values any time? also during charging session?
```

Comment:
@SebaLukas : Please  check if something is missing here

Reply:
Yes, that works.
And there should be no problems with the values changing during a session.

Reply:
ok, then I'll remove the FIXME